tests/examples/custom_agents/travel_agent/config.py

```
@@ -0,0 +1,24 @@
+import sys 
+import os
+from dotenv import load_dotenv
```

Ruff (0.8.2)

`dotenv.load_dotenv` imported but unused

Remove unused import: `dotenv.load_dotenv`

(F401)